### DISCUSSION TITLE
How to deal with seperated images in Acornfile's build/push?

### DISCUSSION THREAD

**Discussion by huapox** [acorn-io/runtime#1720] [General]
Amazing Product first~

**Here with a usage Question**:

How to deal with seperated images in Acornfile's build/push? or How to reuse the Image layer in each container like redis/mysql which already existed in the remote registry.

Currently, when the first build/push, the full layers will need to be sent to the registry.

**Comment by cjellick** [comment]
Thanks for trying it out! I'm going to convert this to a discussion
